command_line_assistant/commands/chat.py

```diff
@@ -74,8 +74,8 @@
 )
 
 #: Max input size we want to allow to be submitted to the backend. This
-#: corresponds to 2KB (2048 bytes)
-MAX_QUESTION_SIZE: int = 2048
+#: corresponds to 32KB (32000 bytes)
+MAX_QUESTION_SIZE: int = 32000
```

Comment:
Super nit: large numbers benefit from underscores to increase legibility.
```suggestion
MAX_QUESTION_SIZE: int = 32_000
```

Reply:
`32 * 1000` would also work.